plugins/git/git_commit_behind

```diff
@@ -0,0 +1,260 @@
+#! /usr/bin/env python3
+
+"""=cut
+=head1 NAME
+
+git_commit_behind - Munin plugin to monitor local git repositories and report
+how many commits behind their remote they are
+
+=head1 NOTES
+
+This plugin is similar to how apt_all works for apt packages.
+
+To be able to check how behind a git repository is, we need to run git fetch.
+To avoid fetching all repos every 5 minutes and slowing down the munin-node,
+the fetch operation is triggered via a cron job.
+
+=head1 REQUIREMENTS
+
+ - Python3
+ - Git
+
+=head1 INSTALLATION
+
+Link this plugin, as usual.
+For example :
+  ln -s /path/to/git_commit_behind /etc/munin/plugins/git_commit_behind
+
+You also need to setup a cron job to trigger the git fetches.
+
+The plugin can be called with an "update" mode to handle the fetches :
+munin-run git_commit_behind update <maxinterval> <probability>
+It will run the fetches randomly (1 in <probability> chances),
+and ensure that it is run at least every <maxinterval> seconds.
+
+For example, you can use the following cron :
+
+# If the git_commit_behind plugin is enabled, fetch git repositories approx.
+# once an hour (12 invocations an hour, 1 in 12 chance that the update will
+# happen), but ensure that there will never be more than two hours
+# (7200 seconds) interval between updates.
+*/5 * * * * root if [ -x /etc/munin/plugins/git_commit_behind ]; then /usr/sbin/munin-run git_commit_behind update 7200 12 >/dev/null; fi
+
+=head1 CONFIGURATION
+
+Use your "/etc/munin/plugin-conf.d/munin-node" to configure this plugin.
+    [git_commit_behind]
+    user root
+    env.git_path /path/to/git
```

Comment:
Maybe mention the default for `git_path`, since most users will not need to override it.

Reply:
I've added details about those settings a few lines below (L51 for `git_path`)